core/tee/tee_rpmb_fs.c

```diff
@@ -588,14 +588,18 @@ static TEE_Result tee_rpmb_req_pack(struct rpmb_req *req,
 			       RPMB_NONCE_SIZE);
 
 		if (rawdata->data) {
-			if (fek)
-				encrypt_block(datafrm[i].data,
-					rawdata->data + (i * RPMB_DATA_SIZE),
-					*rawdata->blk_idx + i, fek, uuid);
-			else
+			if (fek) {
+				res = encrypt_block(datafrm[i].data,
+				                    rawdata->data + (i * RPMB_DATA_SIZE),
+				                    *rawdata->blk_idx + i, fek, uuid);
```

Comment:
coding style: remove the space char and and break lines to not exceed 80char/line(see Travis failure):
```c
				res = encrypt_block(datafrm[i].data,
						    rawdata->data +
						    (i * RPMB_DATA_SIZE),
						    *rawdata->blk_idx + i,
						    fek, uuid);